packages/odyssey-storybook/src/guidelines/Roadmap/roadmapData.tsx

```diff
@@ -31,86 +32,109 @@ export type OdysseyComponent = {
   deliverableTiming: string;
 };
 
-export const columns: DataTableColumn<DataTableRowData>[] = [
-  {
-    accessorKey: "name",
-    header: "Name",
-    enableHiding: false,
-    size: 325,
-  },
-  {
-    accessorKey: "type",
-    header: "Type",
-    enableHiding: true,
-    size: 200,
-  },
-  {
-    accessorKey: "status",
-    header: "Status",
-    size: 200,
-    Cell: ({ cell, row }) => {
-      const statusValue = cell.getValue<string>();
-      const defineValue = row.original.define;
-      const designValue = row.original.design;
-      const developValue = row.original.develop;
-      const severityMap = new Map<
-        string,
-        (typeof statusSeverityValues)[number]
-      >([
-        ["Released", "success"],
-        ["In Labs", "warning"],
-        ["In progress", "default"],
-        ["Not started", "error"],
-      ]);
-      const severity = severityMap.get(statusValue) || "default";
+export const data: OdysseyComponent[] = rawData as OdysseyComponent[];
+
+const severityMap = new Map<string, (typeof statusSeverityValues)[number]>([
+  ["Released", "success"],
+  ["In Labs", "warning"],
+  ["In progress", "default"],
+  ["Not started", "error"],
+]);
+
+const getTooltipText = (
+  defineValue: string,
+  designValue: string,
+  developValue: string,
+): string => {
+  let text = "";
+  if (defineValue === "In progress") {
+    text += "Project definition in progress";
+  }
+  if (["Complete", "In progress"].includes(designValue)) {
+    text += (text ? " " : "") + "Design: " + designValue;
+  }
+  if (["Complete", "In progress"].includes(developValue)) {
+    text += (text ? " " : "") + "Develop: " + developValue;
+  }
+  return text.trim();
+};
```

Comment:
Since this is just the roadmap page, these are optional changes.

In general, I prefer not overloading the `+` operator and especially not doing mutations. I would instead write this with `.concat()` and `return` to avoid mutations.

If it works and looks right, I'm not gonna say much for this PR; I just don't want bad patterns in the codebase in general. I also don't wanna waste time on code for doc pages unless it's going to be too fragile.

Reply:
Good call. Since this is pre-existing and internal only (roadmap page), I am going to tackle this later but I just created a ticket for myself for those changes. There are some other lingering things to explore in the roadmap page as well (the root cause of why it was infinitely refreshing with pagination, search, and filters enabled) so I may work on it when I do that work.